wp-content/themes/rebalance-child-mmn/style.css

```diff
@@ -301,4 +301,38 @@ figcaption {
 
 details summary {
   display:revert;
+}
+
+/* mmn: merites custom css */
+.merites-font {
+  font-family: 'WindsorConDExtBol', sans-serif;
+  margin-bottom: 20px;
+}
+
+/* mmn: css to have the elements of the 4th menu block inline rather than below each other (merites blog/english content) */
```

Comment:
This is clearly a hack, even if a nice one :), let's keep it like this for the moment (so that the feature can be out asap), but let's remember that we should do it properly at some point. I added a new issue for that: https://github.com/mmmnmnm/wordpress/issues/34

Reply:
chatGPT made me do it

Reply:
What? :)
How did you involve AI in this one?

Reply:
I basically prompted everything otherwise I would've needed to learn web dev beforehand :)